src/App/ComponentsDocumentation/components/Buttons/__snapshots__/index.test.js.snap

```diff
@@ -685,6 +695,78 @@ exports[`Documentation: Buttons Developer documentation renders 1`] = `
        attribute on these links (to prevent them from receiving keyboard focus) and use custom JavaScript to disable their functionality.
     </p>
   </Alert>
+  <h2>
+    Loading state
+  </h2>
+  <p>
+    Use the loading state to show users their action is processing by adding the .loading class to the 
+    <CodeTags
+      code="<button>"
+      type="primary"
+    />
+     element.
+  </p>
+  <ComponentPreview
```

Comment:
Some of the buttons have wrong color on the loading circle when hovering over with the mouse. (Applies to the disabled state)